tests/test_changes.list

```diff
@@ -1,98 +1,63 @@
+cpld_control_p8_lnd intel
+cpld_control_p8_mixedmode intel
+cpld_control_pdlib_p8 gnu
 cpld_control_pdlib_p8 intel
-cpld_restart_pdlib_p8 intel
+cpld_control_qr_p8 intel
+cpld_control_sfs intel
+cpld_control_sfs intelllvm
+cpld_dcp_gefs intel
+cpld_debug_gfsv17 intel
+cpld_debug_gfsv17 intelllvm
+cpld_debug_noaero_p8 intel
+cpld_debug_p8 gnu
+cpld_debug_p8 intel
+cpld_debug_pdlib_p8 gnu
+cpld_debug_pdlib_p8 intel
+cpld_decomp_p8 intel
+cpld_mpi_gfsv17 intel
+cpld_mpi_p8 intel
 cpld_mpi_pdlib_p8 intel
-cpld_control_c48_5deg intel
-cpld_warmstart_c48_5deg intel
-cpld_restart_c48_5deg intel
-cpld_control_c24_5deg intel
+cpld_s2sa_p8 intel
+cpld_warmstart_c12_9deg intel
 cpld_warmstart_c24_5deg intel
-cpld_restart_c24_5deg intel
-cpld_control_c24_9deg intel
 cpld_warmstart_c24_9deg intel
-cpld_restart_c24_9deg intel
-cpld_control_c12_9deg intel
-cpld_warmstart_c12_9deg intel
-cpld_restart_c12_9deg intel
-cpld_debug_pdlib_p8 intel
-control_c48 intel
-control_c48.v2.sfc intel
-control_c48_lnd_iau intel
-control_c192 intel
-control_c384 intel
-control_c384gdas intel
-control_p8 intel
-control_p8.v2.sfc intel
-control_p8_ugwpv1 intel
-control_p8_ugwpv1_tempo intel
-control_p8_ugwpv1_tempo_aerosol intel
-control_p8_ugwpv1_tempo_aerosol_hail intel
-control_restart_p8 intel
-control_noqr_p8 intel
-control_restart_noqr_p8 intel
-control_decomp_p8 intel
-control_2threads_p8 intel
-control_p8_rrtmgp intel
-merra2_thompson intel
-merra2_hf_thompson intel
-control_wam intel
-control_p8_faster intel
-control_wam_debug intel
-hafs_regional_atm_thompson_gfdlsf intel
-hafs_regional_1nest_atm intel
-hafs_global_1nest_atm intel
-hafs_global_multiple_4nests_atm intel
-hafs_regional_storm_following_1nest_atm intel
-atm_ds2s_docn_pcice intel
-atm_ds2s_docn_dice intel
-control_p8_atmlnd intel
-control_restart_p8_atmlnd intel
-control_p8_atmlnd_debug intel
-atmwav_control_noaero_p8 intel
-atmaero_control_p8 intel
-atmaero_control_p8_rad intel
-atmaero_control_p8_rad_micro intel
-regional_atmaq intel
-regional_atmaq_canopy intel
-regional_atmaq_debug intel
-control_c48 gnu
-control_p8 gnu
-control_p8_ugwpv1 gnu
-control_wam_debug gnu
-cpld_control_p8 gnu
-cpld_control_nowave_noaero_p8 gnu
-cpld_debug_p8 gnu
-cpld_control_pdlib_p8 gnu
-cpld_debug_pdlib_p8 gnu
-cpld_control_gfsv17 intelllvm
-cpld_debug_gfsv17 intelllvm
-cpld_control_p8 intelllvm
+cpld_warmstart_c48_5deg intel
+datm_cdeps_3072x1536_cfsr intel
+datm_cdeps_bulk_cfsr intel
+datm_cdeps_bulk_gefs intel
+datm_cdeps_ciceC_cfsr intel
+datm_cdeps_control_cfsr_faster intel
+datm_cdeps_control_cfsr gnu
+datm_cdeps_control_cfsr intel
+datm_cdeps_control_cfsr intelllvm
+datm_cdeps_control_gefs intel
+datm_cdeps_debug_cfsr intel
+datm_cdeps_debug_cfsr intelllvm
+datm_cdeps_gfs intel
+datm_cdeps_iau_gefs intel
+datm_cdeps_multiple_files_cfsr intel
+datm_cdeps_mx025_cfsr intel
+datm_cdeps_mx025_gefs intel
+datm_cdeps_stochy_gefs intel
+hafs_regional_storm_following_1nest_atm_ocn_wav_mom6 intel
+cpld_control_gfsv17_iau_intel
```

Comment:
@FernandoAndrade-NOAA Shouldn't `cpld_restart_gfsv17_iau_intel` also be included here?
```suggestion
cpld_control_gfsv17_iau_intel
cpld_restart_gfsv17_iau_intel
```

Reply:
@jiandewang Thanks!